src/devices/openxrheadset/OpenXrInterface.cpp

```diff
@@ -1704,35 +1843,62 @@ void OpenXrInterface::getAllPoses(std::vector<NamedPoseVelocity> &additionalPose
         }
     }
 
-    additionalPoses.resize(numberOfPoses);
+    if (m_pimpl->hand_tracking_supported)
+        numberOfPoses += XR_HAND_JOINT_COUNT_EXT*2;
 
+    additionalPoses.resize(numberOfPoses);
     size_t poseIndex = 0;
 
     auto& head = additionalPoses[poseIndex];
     head.name = "openxr_head";
     head.pose = headPose();
-    head.velocity = headVelocity();
+	head.velocity = headVelocity();
+    head.filterType = PoseFilterType::NONE;
     poseIndex++;
 
     auto& left_arm = additionalPoses[poseIndex];
     left_arm.name = "openxr_left_hand";
-    left_arm.pose = leftHandPose();
-    left_arm.velocity = leftHandVelocity();
+    left_arm.pose = m_pimpl->leftHandPose_;
+    if (m_pimpl->hand_tracking_supported)
+        left_arm.filterType = PoseFilterType::NONE;
```

Comment:
Same as for the head. If the filter gives issues, I would rather add a configuration parameter to disable it. In fact, the ``hand_tracking`` could be supported by the realtime, but we could still use the joysticks for example.